contrib/opportunity/docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM nitlang/nit
+RUN apt-get update && apt-get install gettext libevent-dev libsqlite3-dev -y
+WORKDIR /nit/contrib/opportunity
+RUN sed -i 's/localhost/0.0.0.0/' src/opportunity_web.nit && make
```

Comment:
Instead of a sed, maybe it would be cleaner to skip the makefile and use`nitc -D iface=0.0.0.0:8080`. Or simply change the default to 0.0.0.0!